src/main/java/net/atos/zac/zoeken/converter/ZaakZoekItemConverter.java

```diff
@@ -0,0 +1,182 @@
+        zoekItem.setInitiatorIdentificatie(zgwApiService.findInitiatorForZaak(zaak.getUrl()));
+        zoekItem.setLocatie(convertToLocatie(zaak.getZaakgeometrie()));
+
+        final CommunicatieKanaal kanaal = getCommunicatieKanaal(zaak);
+        if(kanaal != null){
+            zoekItem.setCommunicatiekanaal(kanaal.getNaam());
+        }
+
+        final Group groep = getGroep(zaak);
+        if(groep != null){
+            zoekItem.setGroepID(groep.getId());
+            zoekItem.setGroepNaam(groep.getName());
+        }
+
+        final User behandelaar = getBehandelaar(zaak);
+        if(behandelaar != null){
+            zoekItem.setBehandelaarNaam(getVolledigeNaam(behandelaar));
+            zoekItem.setBehandelaarGebruikersnaam(behandelaar.getId());
+        }
+
+        if(zaak.getVerlenging() != null){
+            zoekItem.setIndicatieVerlenging(true);
+            zoekItem.setDuurVerlenging(String.valueOf(zaak.getVerlenging().getDuur()));
+            zoekItem.setRedenVerlenging(zaak.getVerlenging().getReden());
+        }
+
+        if(zaak.getOpschorting() != null){
+            zoekItem.setRedenOpschorting(zaak.getOpschorting().getReden());
+        }
+
+        final Zaaktype zaaktype = ztcClientService.readZaaktype(zaak.getZaaktype());
+        zoekItem.setZaaktypeIdentificatie(zaaktype.getIdentificatie());
+        zoekItem.setZaaktypeOmschrijving(zaaktype.getOmschrijving());
+        zoekItem.setZaaktypeUuid(UriUtil.uuidFromURI(zaaktype.getUrl()).toString());
+
+        final Status status = zrcClientService.readStatus(zaak.getStatus());
+        final Statustype statustype = ztcClientService.readStatustype(status.getStatustype());
+        zoekItem.setStatusNaam(statustype.getOmschrijving());
+        zoekItem.setStatusEindstatus(statustype.getEindstatus());
+        zoekItem.setStatusToelichting(status.getStatustoelichting());
+        zoekItem.setStatusToekenningsdatum(toSolrDate(status.getDatumStatusGezet()));
+
+        if(zaak.getResultaat() != null) {
+            final Resultaat resultaat = zrcClientService.readResultaat(zaak.getResultaat());
+            if (resultaat != null) {
+                Resultaattype resultaattype = ztcClientService.readResultaattype(resultaat.getResultaattype());
+                zoekItem.setResultaatNaam(resultaattype.getOmschrijving());
+                zoekItem.setResultaatToelichting(resultaat.getToelichting());
+            }
+        }
+
+        return zoekItem;
+    }
+
+
+    private String convertToLocatie(final Geometry zaakgeometrie) {
+        //todo
+        return null;
+    }
+
+    private User getBehandelaar(final Zaak zaak) {
+        return zgwApiService.findBehandelaarForZaak(zaak.getUrl())
+                .map(betrokkene -> betrokkene.getBetrokkeneIdentificatie().getIdentificatie())
+                .map(id -> flowableService.readUser(id))
+                .orElse(null);
+    }
+
+    private String getVolledigeNaam(final User user) {
```

Comment:
Het bepalen van de volledige naam is logica die bij het User object moet zitten.